server/apiGatewayDiscovery.ts

```diff
@@ -149,7 +149,13 @@ const getApiGateway = (
 				shouldNotLogBody?: boolean,
 			) =>
 			async (req: express.Request, res: express.Response) => {
-				const isTestUser = req.header(MDA_TEST_USER_HEADER) === 'true';
+				const testUserHeader = req.header(MDA_TEST_USER_HEADER);
+				if (!['true', 'false'].includes(testUserHeader ?? '')) {
```

Comment:
could we change this to something like the following ... I wonder if it's a bit more readable:

```
if (testUserHeader === undefined || !['true', 'false'].includes(testUserHeader)) {
```